src/main/java/uk/gov/justice/hmpps/prison/repository/jpa/model/AssessmentEntry.java

```diff
@@ -0,0 +1,39 @@
+package uk.gov.justice.hmpps.prison.repository.jpa.model;
+
+import lombok.*;
```

Comment:
Meh, wildcard imports aren't allowed in the project.

Reply:
I have no idea how that got there.. fixing